src/javascript/audio.js

```diff
@@ -86,3 +127,25 @@ export default class Audio {
     this.player?.rate(rate);
   }
 }
+
+function shuffle(array) {
+  //ref https://stackoverflow.com/questions/2450954/
+
+  let currentIndex = array.length,
+    temporaryValue,
+    randomIndex;
```

Comment:
```suggestion
  let currentIndex = array.length;
  let randomIndex = 0;
```
Don't even need the `temporaryValue`, see ES6 array-destructing below 🧀